x/cash/decorator.go

```diff
@@ -142,7 +142,7 @@ func (d FeeDecorator) extractFee(ctx weave.Context, tx weave.Tx, store weave.KVS
 
 	}
 	if !fee.IsGTE(cmp) {
-		return nil, ErrInsufficientFees(*fee)
+		return nil, errors.ErrInsufficientAmount.Newf("fees %#v", fee)
```

Comment:
fees too low %#v (or %s) ?

Reply:
Hmm, why? I thought it was about not repeating ourselves, it's already "Insufficient amount: fees 'format'", which implies too low

Reply:
good point